build-system/tasks/visual-diff/helpers.js

```diff
@@ -191,8 +191,12 @@ async function waitForElementVisibility(page, selector, options) {
 
     for (const elementHandle of await page.$$(selector)) {
       const boundingBox = await elementHandle.boundingBox();
+      const fullyOpaque = (await getElementOpacity(elementHandle)) === '1';
```

Comment:
Sometimes an element can naturally have lower opacity in the document's stylesheet. I think this is generally ok - will just be a limitation of the diffing system. We just need to call it out in the docs.

Reply:
That's a great point, we should definitely call it out in the docs.

Reply:
I'm just still feeling like this is prone to come back and bite us eventually, so I wanna be extra careful here.
@rileyajones I assume this is meant to handle the case of animated transitions? In general Percy forces all animations to be 0 seconds long on server side, if this is only supposed to handle CSS transitions perhaps we can force the same kind of constraint on our visual diffs instead of awaiting on the opacity (basically, treat the cause and not the effect) - is this the case you're trying to handle?